central/notifiers/email/email_test.go

```diff
@@ -48,6 +51,107 @@ func TestEmailMsgWithAttachment(t *testing.T) {
 
 	assert.Contains(t, msgStr, base64.StdEncoding.EncodeToString(attachBuf.Bytes()))
 	assert.Contains(t, msgStr, "<div>\r\nHow you doin'?\r\n</div>\r\n")
+
+	lastBoundary, expectedFinalBoundary, err := obtainLastAndExpectedBoundaryString(msgStr)
+	if err != nil {
+		t.Fatalf("Failed to compile regex for boundary: %v", err)
+	}
+	assert.Equal(t, expectedFinalBoundary, lastBoundary)
+}
+
+func obtainLastAndExpectedBoundaryString(msgStr string) (string, string, error) {
+	// Obtain boundary and verify the close delimiter
+	regex := regexp.MustCompile(` boundary="([^"]+)"`)
+	// Regex will provide two values [boundary="boundaryValue", boundaryValue]
+	boundary := regex.FindStringSubmatch(msgStr)
```

Comment:
Use `regex.FindString(msgStr)` . That will match `boundary="boundaryValue"`. Then you can split by '=' and remove double quotes from splitResults[1] to get `boundaryValue`. That will make it more clear to understand what is being selected here.

Reply:
Thanks for this suggestion, this makes the code more clear.